src/widget/chatformheader.cpp

```diff
@@ -44,44 +30,43 @@ const short BUTTONS_LAYOUT_HOR_SPACING = 4;
 const QString STYLE_PATH = QStringLiteral("chatForm/buttons.css");
 
 const QString STATE_NAME[] = {
-    QString{},
-    QStringLiteral("green"),
-    QStringLiteral("red"),
-    QStringLiteral("yellow"),
-    QStringLiteral("yellow"),
+    QString{},                //
+    QStringLiteral("green"),  //
+    QStringLiteral("red"),    //
+    QStringLiteral("yellow"), //
+    QStringLiteral("yellow"), //
```

Comment:
why?

Reply:
So clang-format doesn't put some of them on the same line.

Reply:
meh, fine